src/components/Register.js

```diff
@@ -40,7 +40,8 @@ const Register = () => {
       }
       if (user) {
         setUser(user);
-        history.push("/login");
+        setCurrentUsername(user.username);
+        history.push("/authenticated");
```

Comment:
Do we need to add those lines even though a user just registers, not logs in? Did you decide to implement if a user registers then logged in at the same time?